### PR TITLE
Add API site id and API robot id in settings

### LINKED ISSUE #55: Load config variables
**Describe the new feature you would like to see**
We need to load some static ids like siteId and robotId from config file
Use the settings.py and settings.env for this

### PR DESCRIPTION
**Pull request by monkeyysups**
Closes #55

### CODE REVIEW

**Comment by monkeyysups** [comment]
Alle andre variabler hentes fra siteType i API

File: src/isar_exr/config/settings.py [hunk 1]
```diff
@@ -21,6 +21,10 @@ class Settings(BaseSettings):
         default="https://login.energy-robotics.com/api/loginApi"
     )
 
+    # API variables
+    API_SITE_ID: str = Field(default="6287873b6cc0c0205c56ea58")
+    API_ROBOT_ID: str = Field(default="628787246cc0c0205c56e88e")
```
Comment (GodVenn) [inline src/isar_exr/config/settings.py:26]:
```suggestion
    ROBOT_API_SITE_ID: str = Field(default="6287873b6cc0c0205c56ea58")
```

Comment (GodVenn) [inline src/isar_exr/config/settings.py:26]: I think this is already defined on line 9

**Review by GodVenn — CHANGES_REQUESTED** [review]
The ROBOT_ID is already added, otherwise rename the site to match naming convention of others

File: src/isar_exr/config/settings.py (hunk 1 above)
Comment (monkeyysups) [reply src/isar_exr/config/settings.py]: done

Comment (monkeyysups) [reply src/isar_exr/config/settings.py]: Updated value on line 9

File: src/isar_exr/config/settings.py [hunk 2]
```diff
@@ -6,7 +6,10 @@
 
 class Settings(BaseSettings):
     # The ID of the robot in the EXR database
-    ROBOT_EXR_ID: str = Field(default="606db4aec70b7c3a944b89d9")
+    ROBOT_EXR_ID: str = Field(default="628787246cc0c0205c56e88e")
+
+    # The ID of KAA site in the EXR database
+    ROBOT_API_SITE_ID: str = Field(default="6287873b6cc0c0205c56ea58")
```
Comment (GodVenn) [inline src/isar_exr/config/settings.py:12]:
```suggestion
    # The ID of the relevant site in the EXR database (default=KAA)
    ROBOT_EXR_SITE_ID: str = Field(default="6287873b6cc0c0205c56ea58")
```

Comment (monkeyysups) [reply src/isar_exr/config/settings.py]: done

**Pull request merged**